CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- `KubernetesWorker` for executing flow runs within Kubernetes jobs - [#40](https://github.com/PrefectHQ/prefect-kubernetes/pull/40)
```

Comment:
question: are we launching this worker as fully supported, or will we go through a beta period or similar? what sort of backward compatibility commitments are we making with this?

Reply:
Nice, thanks for the clarification!